torch/onnx/symbolic_opset9.py

```diff
@@ -2423,3 +2423,37 @@ def take(g, self, index):
     out = index_select(g, self_flattened, 0, index)
     out = reshape_as(g, out, index)
     return out
+
+
+def _kl_div_log_target(g, input, target):
```

Comment:
Are `_kl_div_log_target` and `_kl_div_non_log_target` names of the symbolics coming from aten itself or these just helper functions? If it is the latter I would suggest naming them differently, possibly with the word `_impl` in the name, to distinguish them from real symbolic entry points in this file. E.g. `_kl_div_log_target_impl` and `_kl_div_non_log_target_impl`.